### ISSUE TITLE
Index not updating on file create/delete

### ISSUE THREAD

**Issue by vredesbyyrd** [ozeidan/gosearch#18]
Hi, I rely on `gosearch` as my main search tool on the daily. Thanks for sharing it, its great tool that should have more exposure imho.  

I cannot be sure when this issue started, but the index is no longer "automagically"  updating when files are created, deleted etc. I am also unsure how to go about debugging this.  `systemctl status gosearch` does show `fanotify.go` and `indexer.go` are active and receiving events, but lets say we create file `$HOME/USER/HelloWorld.sh`, querying said file does not return a result. Re-indexing manually, `systemctl restart gosearch` and running the query again correctly returns `$HOME/USER/HelloWorld.sh`. 

**Environment info:** 

Kernel: `5.12.1.arch1-1` 
Go: `2:1.16.4-1` 

Not sure the `atime` mount option is relevant, but `ext4` filesystem is mounted with `relatime`.  

If you have any thoughts on this issue I would love to hear them! 

Thanks for your time.

**Comment by ozeidan** [comment]
Hello, glad to hear that gosearch is still being used. I'm currently not using Linux so it's possible that there has been a change in the kernel API that went unnoticed. Also I won't be able to look immediately but hopefully in the next couple of days 👍🏻 I agree it's a nice project, I think I'll put some more time into it soon

**Comment by vredesbyyrd** [comment]
>  ...it's possible that there has been a change in the kernel API that went unnoticed. 

Yeah I was thinking that might be the case to. Ill take a look at change logs and see if anything jumps out to me.  

> Also I won't be able to look immediately but hopefully in the next couple of days 👍🏻 

No sweat, i'm just happy it may get some attention, thank you!  

> I agree it's a nice project, I think I'll put some more time into it soon 

That would be quite awesome. `gosearch` + `fzf` and its preview window for image/file previews is a beautiful thing.
